includes/class-wc-cache-helper.php

```diff
@@ -136,7 +136,7 @@ public static function get_transient_version( $group, $refresh = false ) {
 		$transient_value = get_transient( $transient_name );
 
 		if ( false === $transient_value || true === $refresh ) {
-			self::delete_version_transients( $transient_value );
+			self::queue_delete_version_transients( $transient_value );
```

Comment:
There is still [one failing unit test](https://travis-ci.org/woocommerce/woocommerce/jobs/392314671). It seems to be caused by this change, as reverting this line to instantly delete makes it pass again. 

If there is potential for this to cause issues, maybe have the deferred transaction delete only happen if a defined constant is true (e.g. `WC_DEFER_TRANSACTION_DELETE`)?

Reply:
Tests run so quick I assume it's because `time()` is returning the same value. Thats a bug.